MojioSDK/mojiosdksrc/src/main/java/io/moj/mobile/android/sdk/MojioClient.java

```diff
@@ -291,18 +250,13 @@ public void onFailure(ResponseError error) {
         }
 
         String entityPath = String.format("Login/%s?secretKey=%s&userOrEmail=%s&password=%s&minutes=%d",
```

Comment:
if the exception gets thrown it'll just use the unencoded username/password. UnsupportedEncodingException really isn't something that'll ever get thrown but, if for some reason it does, the call will only fail if the user's username or password contains a symbol that must be encoded so there's a chance we're still OK.

In reality we should never pass login information as a URL parameter - I'm going to open a TP bug to have this looked at as it is definitely a security issue.